src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -189,8 +207,47 @@ class BaseOptionsSelector extends Component {
         return defaultIndex;
     }
 
+    /**
+     * When pagination is enabled,
```

Comment:
Is pagination something that can be enabled? I thought it's just the default behavior of the component.

Reply:
It is an echo from the past when it was a prop. Good catch!

Reply:
Done - 49154ff.